crates/solidity/outputs/npm/package/src/generated/index.d.ts

```diff
@@ -519,7 +519,6 @@ export namespace language {
     constructor(version: string);
     get version(): string;
     static supportedVersions(): Array<string>;
-    scan(lexicalContext: LexicalContext, input: string): kinds.TokenKind | null;
     parse(kind: kinds.RuleKind, input: string): parse_output.ParseOutput;
```

Comment:
I suggest adding a changeset about the removed API.

Reply:
Done in https://github.com/NomicFoundation/slang/pull/728